CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ All notable changes to [bpmnlint-plugin-camunda-compat](https://github.com/camun
 
 ___Note:__ Yet to be released changes appear here._
 
+* `DEPS`: bump `zeebe-bpmn-moddle@1.7.0`
+* `FEAT`: support task listeners ([#181](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/181))
+* `FEAT`: add `no-duplicate-task-listeners` rule
```

Comment:
There was a decision to not reject duplicate listeners, but we can still show a warning. Cf. https://github.com/camunda/camunda-modeler/issues/4590#issuecomment-2446717471

Reply:
I'd say let's not add this rule to the compat project. We can open a discussion on a different set of rules. Perhaps bpmnlint-plugin-camunda is the right place for the best practices?

Reply:
As discussed, I am removing this completely